src/main/java/org/molgenis/vcf/decisiontree/filter/model/NestedField.java

```diff
@@ -0,0 +1,41 @@
+package org.molgenis.vcf.decisiontree.filter.model;
+
+import static java.util.Objects.requireNonNull;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+import org.molgenis.vcf.decisiontree.runner.info.NestedInfoSelector;
+
+@Value
+@NonFinal
+public class NestedField extends Field {
```

Comment:
Would it make more sense to have one `Field` class for all fields since the `FieldType` already states that it is a nested `Field`. I have troubles getting my head about this construct.